src/sentry/sentry_metrics/indexer/postgres/models.py

```diff
@@ -45,7 +45,7 @@ def get_next_values(cls, num: int) -> Any:
 
 
 class BaseIndexer(Model):
-    string = models.CharField(max_length=200)
+    string = models.CharField(max_length=MAX_INDEXED_COLUMN_LENGTH)
```

Comment:
Do we not need a migration to go along with this? assuming we did cause docs say it's checked on the db level https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.CharField.max_length but maybe im wrong here

Reply:
`MAX_INDEXED_COLUMN_LENGTH` is defined to be 200 [here](https://github.com/getsentry/sentry/pull/53493/files#diff-eb6f4d8deb01417b0b938fa35be53b4ba1e5c9508530c793765ebe4115006632R14). The actual value hasn't changed. If the actual value was being changed I am sure you would be correct that a migration is needed. I am not sure whether migration is needed if the value is same as before.

Reply:
I checked locally by creating a sample django app and modifying the max_length from a value to a constant. Then ran makemigrations in both cased to see if there is any differences. Both generate the same output. So this shouldn't be a problem.